client/tinychain/ml/mlr.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
you don't need this shebang line unless you expect this script to be executed by itself (which it shouldn't be since this is a single library)